hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/TokenUpdateHandlerParityTest.java

```diff
@@ -145,12 +152,23 @@ void tokenUpdateReplacingAdminKey() throws PreCheckException {
     void tokenUpdateWithSupplyKeyedToken() throws PreCheckException {
         final var txn = txnFrom(UPDATE_WITH_SUPPLY_KEYED_TOKEN);
         final var context = new FakePreHandleContext(readableAccountStore, txn);
+        final var lowPriorityKeyList = Key.newBuilder()
```

Comment:
NIT: could refactor to declare lowPriorityKeyList and thresholdKey once and avoid duplication

Reply:
@thomas-swirlds-labs 
The `lowPriorityKeyList` contains different key for each keys and there are only 4 occurrences of these types of tests... do you mean to declare them inside `setUp()` which will be executed before each of the test cases?